config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/user_id/posts', to: 'posts#index'
+  get '/users/user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [OPTIONAL] Kindly write the routes in this way:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

as it's cleaner and easier to read and maintain.

Reply:
Thank @gemmen29 for review, I will make these changes .